app/views/playlists/show.html.erb

```diff
@@ -31,34 +31,47 @@
 
 <br>
 
+<%#= javascript_tag("playTracks(#{@playlist.hold_time}, #{@playlist.tracks_for_js})") %>
+
 <script type="text/javascript">
```

Comment:
This async function is not giving me what I want. Ideally, 
1: loop through all audio file names in the loop and play the file. Then wait for `holdTime` amount of seconds. So, play track, then hold. 
2: after finishing the whole array, play the final closing track

ex:
```
play track1
hold for 30 seconds
play track2
hold for 30 seconds
play track3
hold for 30 seconds
[end loop]
play ending track
```

What it's actually doing:
```
hold for 30 seconds <-- do not want!
play track1
hold for 30 seconds
play track2
hold for 30 seconds
play track3
hold for 30 seconds
play ending track (as the last track in the array)
```

@tomrich82 any ideas?

Reply:
I found a solution using a `forEach` loop with a closure: https://stackoverflow.com/a/43411406/5009528